tests/installation/select_patterns.pm

```diff
@@ -107,6 +102,11 @@ sub run {
             };
             assert_screen 'current-pattern-selected', 5;
         }
+
+        # stick to the default patterns
+        if (get_var('PATTERNS', '') =~ /default/) {
+            $needs_to_be_selected = $selected;
```

Comment:
Yeah, true. The elsif is just a micro-optimation - perl will not evaluate the if in case the previous if matched.

But my point is that you don't need the assignment if you stick with elsif - but without the assignement the logic looks rather strange. So just remove the els and we should be fine  :)